plugins/http_plugin/http_plugin.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <boost/asio.hpp>
 #include <boost/optional.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
I don't think #1011 will be merged before this, I'll resolve the conflict when PR #1011 is ready to merge. It would cause link error if I don't resolve the problem at the time.